includes/class-wc-payments-utils.php

```diff
@@ -728,16 +728,16 @@ public static function get_last_refund_from_order_id( $order_id ) {
 	}
 
 	/**
-	 * Helper function to check whether the user is either in the PO experiment, or has manually enabled PO via the dev tools.
+	 * Helper function to check whether to show default progressive onboarding experience or as an expection disable it (if specific constant is set) .
 	 *
 	 * @return boolean
 	 */
 	public static function should_use_progressive_onboarding_flow(): bool {
-		if ( self::is_in_progressive_onboarding_treatment_mode() || WC_Payments_Features::is_progressive_onboarding_enabled() ) {
-			return true;
+		if ( defined( 'WCPAY_DISABLE_PO' ) && WCPAY_DISABLE_PO ) {
```

Comment:
Nitpick: Can we call this flag something like `WCPAY_DISABLE_NEW_ONBOARDING` or `WCPAY_DISABLE_NEW_KYC`?

I think for a long time we've been using "Progressive Onboarding" to mean both the new KYC and the actual PO experience, and it causes some confusion when talking to other stakeholders about it, so this seems like a good opportunity to separate the two things. 🙂

Reply:
Also, we could change the function name to `should_use_new_onboarding_flow`. 🙂

Reply:
I like `WCPAY_DISABLE_NEW_ONBOARDING` 👍 
And function naming too 🥇 
Thank you!

Reply:
Done in https://github.com/Automattic/woocommerce-payments/pull/7773/commits/6b4b3d4536b8fa13538dda3b62e7f76ad0f469b2